lib/active_storage_blob_content_types.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+##
+# An extension that will be included into the ActiveStorage::Blob class
+module ActiveStorageBlobContentTypes
```

Comment:
What if we.... override `analyze` to make the content type update?

https://github.com/rails/rails/blob/master/activestorage/app/models/active_storage/blob/analyzable.rb

So, something like...

```
def analyze
  update! metadata: metadata.merge(extract_metadata_via_analyzer), content_type: marc_content_type
end

def marc_content_type
  case analyzer.identify
  when :marc21 then 'application/marc'
  when :marcxml then 'application/marcxml+xml'
  else content_type
  end
end
```

?